### DISCUSSION TITLE
Could someone help me

### DISCUSSION THREAD

**Discussion by ManiadT-T** [i3/i3#5703] [Q&A] [answered]
In my i3 (gaps) config file I added a line to run a bash script to execute midnight commander 

bindsym $mod+m exec Mythings/MyBash/mc/Gomc.sh 

the problem is that when I press my mod key + m nothing happens yet if I open my terminal and run Mythings/MyBash/mc/Gomc.sh midnight commander opens.

inside the Gomc.sh there is:

EDITOR=nvim mc

Does someone know why i3 wont execute my bash script? Thanks!

**Comment by okraits** [comment]
Things to check:

1. Does the binding `$mod+m` execute other commands, for example `exec i3-nagbar -m foo`?
2. Do you use an absolute path to the script you are trying to execute?
3. Is your script executable and does it have a shebang at the top?

**Comment by orestisfl** [comment, marked as answer]
When you run your binding from your config, there is no terminal to run a TUI in. You will need to figure out how to run a new process with your preferred terminal. For example:

```
gnome-terminal -- /full/path/to/script.sh
```
